### PR TITLE
check image scheme

### PR DESCRIPTION
**Pull request by lun3322**
image url maybe not have scheme

### CODE REVIEW

File: goose3/extractors/images.py [hunk 1]
```diff
@@ -412,3 +414,10 @@ def load_customesite_mapping(self):
         for line in lines:
             domain, css = line.split('^')
             self.custom_site_mapping.update({domain: css})
+    
+    def add_schema_if_none(self, src):
+        src_test = parse_url(src)
```
Comment (cdrappi) [inline goose3/extractors/images.py]: `parse_url` is not imported here

Comment (lababidi) [reply goose3/extractors/images.py]: @cdrappi why not just import parse_url?

Comment (cdrappi) [reply goose3/extractors/images.py]: which one? i see no `parse_url` used anywhere else in the project

Comment (lababidi) [reply goose3/extractors/images.py]: good point. I assumed the author knew what they were doing.

Could you decipher what they were trying to do and contribute?

Comment (cdrappi) [reply goose3/extractors/images.py]: @lababidi i believe this is what the author intended to do. https://github.com/goose3/goose3/pull/11

disclosure, i have little skin in the game as my usecase has no need for image stuff

**Pull request merged**